cpp/include/cudf/io/json.hpp

```diff
@@ -53,6 +53,16 @@ struct schema_element {
    * @brief Allows specifying this column's child columns target type
    */
   std::map<std::string, schema_element> child_types;
+
+  /** @brief Allows specifying the order of the columns
+   */
+  std::optional<std::vector<std::string>> column_order;
+
+  /** @brief Returns the size of the child_types map
+   *
+   * @return size_t The size of the child_types map
+   */
+  size_t size() const { return child_types.size(); }
```

Comment:
`size` is ambiguous here, since it means number of children. Should be `num_children()` instead.

Reply:
This is added to allow std::visit function work. It's not used anywhere else.